.github/workflows/create-release.yml

```diff
@@ -30,17 +30,21 @@ jobs:
         fi
         echo "version=${version}" >> "$GITHUB_OUTPUT"
 
-    - name: Wait for rootfs docker image
+    - name: Auth GitHub CLI
+      run: echo "${{ secrets.GITHUB_TOKEN }}" | gh auth login --with-token
```

Comment:
GH CLI will automatically log in using the `GH_TOKEN` environment variable. So you can also replace this `gh auth login --with -token` with setting the env var at the workflow/job level, e.g. `env: GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}`